assets/js/components/ClusterDetails/ChecksResults.jsx

```diff
@@ -126,7 +126,7 @@ export const ChecksResults = () => {
 
     pageContent = lastExecution.map(
       ({ _cluster_id, host_id, reachable, msg }, idx) => (
-        <div key={idx} className="flex flex-col">
+        <div key={idx} className={`flex flex-col ${idx != (cluster.hosts_executions.length - 1) ? "mb-8" : ""}`}>
```

Comment:
You could use `classNames` (`import classNames from 'classnames'`) and just do `classNames('flex', 'flex-col', {'mb-8': idx != (cluster.hosts_executions.length - 1)})`

Reply:
Cool, I can make that change.